src/main/java/org/folio/services/validator/registry/ValidatorRegistryService.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.services.validator.registry;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.ProxyGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Validator Registry service interface, performs CRUD operations on Rule entity
+ */
+@ProxyGen
+public interface ValidatorRegistryService {
+
+  static ValidatorRegistryService create(Vertx vertx) {
+    return new ValidatorRegistryServiceImpl(vertx);
+  }
+
+  static ValidatorRegistryService createProxy(Vertx vertx, String address) {
+    return new ValidatorRegistryServiceVertxEBProxy(vertx, address);
+  }
+
+  @Fluent
+  ValidatorRegistryService getAllTenantRules(String tenantId, Handler<AsyncResult<JsonObject>> asyncResultHandler);
+
+  @Fluent
+  ValidatorRegistryService createTenantRule(String tenantId, JsonObject validationRule, Handler<AsyncResult<JsonObject>> asyncResultHandler);
+
+  @Fluent
+  ValidatorRegistryService updateTenantRule(String tenantId, JsonObject validationRule, Handler<AsyncResult<JsonObject>> asyncResultHandler);
+
+  @Fluent
+  ValidatorRegistryService getTenantRuleByRuleId(String tenantId, String ruleId, Handler<AsyncResult<JsonObject>> asyncResultHandler);
+
+  @Fluent
+  ValidatorRegistryService getActiveRulesByType(String tenantId, String type, Handler<AsyncResult<JsonObject>> asyncResultHandler);
```

Comment:
I see the discrepancy between method naming (Active) and rule state (Enabled/Disabled).
Let's define only one way of naming rule state and method name. 
I would propose to reword "getActiveRulesByType" to "getEnabledRulesByType"

Reply:
Done